docs/binary-rendering.md

```diff
@@ -52,7 +52,7 @@ Example usage with `HttpServletResponse`:
 === "Kotlin"
 
     ```kotlin linenums="1"
-    val output = new Utf8ByteOutput()
```

Comment:
Good catch!

I'm actually trying to find the time to externalize the code samples in `.java`/`.kt` files that are compiled. It is [supported by MkDocs](https://squidfunk.github.io/mkdocs-material/reference/code-blocks/#embedding-external-files), and it would be easier to maintain. We can also externalize the template examples.